java/src/main/java/com/cybersource/ws/client/SecurityUtil.java

```diff
@@ -127,7 +127,7 @@ private static void readAndStoreCertificateAndPrivateKey(MerchantConfig merchant
         }
         
         try {
-            merchantKeyStore.load(new FileInputStream(merchantConfig.getKeyFile()), merchantConfig.getKeyPassword().toCharArray());
+            merchantKeyStore.load(new FileInputStream(merchantConfig.getKeyFile()), merchantConfig.getKeyPassword());
```

Comment:
I understand that, this is not related to your task but can you please close this FileInputStream.

Reply:
closed the File input stream